app/controllers/collections_controller.rb

```diff
@@ -144,6 +144,7 @@ def set_collection
       @native_id = meta['native-id']
       @provider_id = meta['provider-id']
       @num_granules = meta['granule-count']
+      @num_variables = meta.fetch('associations', {}).fetch('variables', []).count
```

Comment:
I think this can dig.

Reply:
@ryanmiller-1 `.count` raises an error when it's evaluated on a nil object (ie. `nil.count`) and `.dig` returns a nil object if the keys passed aren't found (or if the desired value happens to be nil), but `.fetch` allows that optional parameter to be passed which helps with the error handling